com.unity.netcode.gameobjects/Tests/Runtime/NetworkAnimator/NetworkAnimatorTests.cs

```diff
@@ -18,10 +18,21 @@ public class NetworkAnimatorTests : BaseMultiInstanceTest
         private Animator m_PlayerOnServerAnimator;
         private Animator m_PlayerOnClientAnimator;
 
+
```

Comment:
so I'm not sure how changes in this test file actually tests against the fix proposed here?

Reply:
Before, we just called `Setup` and set things up in host mode.  Host mode passed the old code with the bug, because the bug only shows up in server mode.  The new test code now supports host and server mode and would expose the bug (if it wasn't fixed)